charts/pgbouncer/values.yaml

```diff
@@ -0,0 +1,92 @@
+#############################
+#  Pgbouncer Configuration  #
+#############################
+image:
+  repository: quay.io/astronomer/ap-pgbouncer-krb
+  tag: 1.1.17-1
+  pullPolicy: IfNotPresent
+
+internalPort: 5432
+servicePort: 5432
+
+podDisruptionBudget:
+  enabled: false
+  config: {}
+
+networkPolicies:
+  enabled: true
+
+env: []
```

Comment:
there is no place to add extraenv as per values.yaml  - all are currently hardcoded in helper.yaml

Reply:
I fixed this in https://github.com/astronomer/astronomer/pull/1545/commits/376ee8b3ed18968db3f107ea4358c1f44ae59b06

Reply:
Maybe a better way to do this would be to put the default values into the `env` section.

Reply:
TBH: it's not a default values. It's required values.